Entity/BannerRepository.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace WebtownPhp\BannerBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+/**
+ * BannerRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class BannerRepository extends EntityRepository
+{
+    /**
+     * @param string $placeName
+     *
+     * @return array
+     */
+    public function getBannersForPlace($placeName)
+    {
+        $query = $this->_em->createQuery(
```

Comment:
Ezt `@todo`-zd be, hogy később esetlegesen optimalizáljuk, illetve benchmarkoljuk mondjuk 10 millió sorra. Az a baj, hogy az `OR`-t tartalmazó lekérdezések, 10-100-szor is lassabban futhatnak le, mint ha mondjuk 2 külön query-be kerülnének, ahogy nő a tábla méret. Bár nyilván ez inkább a join-olt lekérdezéseknél számottevő.